content/broker/broker-transition/usage/rest/_index.nb.md

```diff
@@ -0,0 +1,349 @@
+---
+title: Brukerveiledning REST
+linktitle: Brukerveiledning REST
+description: Forskjell i bruk av REST operasjoner mellom Altinn 2 og Altinn 3 overførte tjenester.
+tags: [Løsning, broker, guide, overgang]
+toc: false
+weight: 1
+---
+
+## Forskjell i bruk mellom eksisterende Altinn 2 REST-operasjoner og Altinn 2 REST-operasjoner som har overføres til Altinn 3.
+
+For standard bruk av Altinn 2 REST-operasjoner, se [Altinn Docs] (https://Altinn.github.io/docs/api/rest/formidling/)
+
+For alle overgangstjenester Altinn 3 vil kvitteringer være pseudo-mottaker generert fra Altinn 3 fil metadata. Dette betyr at alle kvitteringer som returneres vil være pseudo-kvitteringer, og alle kvitteringsid vil være 0.
+
+Hvis brukssaken din krever bruk av kvittering, kan du sende inn en endringsforespørsel. 
+
+## Outbox (Sender)
+
+Restoperasjoner for Broker i Altinn 2 er logisk delt mellom avsender og mottaker (Outbox / inbox).
+
+### Initialize and Upload
+
+Altinn 2 REST-operasjoner kombinerer initiering og last opp til samme operasjon.
+Operasjonen består av en postforespørsel med en fil som en binært body og filnavn og BrokerServiceDescription lagt til som parameter.
+
+Header
+```HTTP
+POST https://www.altinn.no/api/{who}/brokerservice/outbox?fileName={fileName}&brokerServiceDescription={"ServiceCode":"{serviceCode}","ServiceEditionCode":{serviceEdition},"Recipients":["{recipient1}","{recipient2}"],"SendersReference":"{sendersReference}}","Properties":{"{name1}":"{value1}", "{name2}":"{value2}"},"FileList":null}
+accept: application/json
+ApiKey: myKey
+Content-Type: application/zip
+```
+
+BrokerServiceDescription example
+
+```JSON
+{
+  "ServiceCode": "5678",
+  "ServiceEditionCode": 1,
+  "SendersReference": "somereference",
+  "Recipients": [
+    "974760673"
+  ],
+  "Properties": {
+    "somethingservicespecific": "somevalue",
+    "somethingelseservicespecific": "someothervalue"
+  },
+  "FileList": null
+}
+```
+
+#### Forskjeller
+
+Overgangstjenesten bruker ikke FileList propertyen.Ingen endring er nødvendig fra sluttbrukerimplementeringen, men en nullverdi vil bli akseptert, og eventuelle sendte verdier vil bli ignorert av overgangstjenesten.
```

Comment:
should be fixed. There's 3 references to "Altinn 3 Overgangs-tjenester", but I think those are fine, since they're specific to services that are transitioned to Altinn 3.